istring.cc

```diff
@@ -54,7 +55,7 @@ int strncasecmp(const char *s1, const char *s2, uint32 length) {
 }
 
 int strcasecmp(const char *s1, const char *s2) {
-	return strncasecmp(s1, s2, 2147483647);
+	return strncasecmp(s1, s2, std::numeric_limits<int32_t>::max());
```

Comment:
Hm. I have no idea why it was just not using strcasecmp instead; could you please change to just remove the third parameter and include and use strcasecmp instead?

Reply:
You mean to use the library function? My guess it's not been use before because it's POSIX, not standard library.

Reply:
Both strncasecmp and strcasecmp are POSIX functions (POSIX.1-2001, according to the man page), so no difference there.

Reply:
Yeah, but see the whole file. `strncasecmp` is actually implemented some lines above my fix (`istring.cc:34`). The call refers to `Pentagram::strncasecmp`, not the global one. Unless Exult's implementation can be ifdefed-away and built only on Windows.